### PR TITLE
Add New Board to compilation output

### PR DESCRIPTION
**Pull request by sbtoonz**
This PR adds build output for 
https://oshwlab.com/b.brazdziunas/scg-ecu_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy_copy

board by @opelpanfan

### CODE REVIEW

File: firmware/config/boards/48way/board_configuration.cpp [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+	// Throttle #1
+	// PWM pin
+	//engineConfiguration->etbIo[0].controlPin = Gpio::B8;
+	// DIR pin
+	//engineConfiguration->etbIo[0].directionPin1 = Gpio::B9;
+	// Disable pin
+	//engineConfiguration->etbIo[0].disablePin = Gpio::B7;
+	// Unused
+	//engineConfiguration->etbIo[0].directionPin2 = Gpio::Unassigned;
```
Comment (rusefillc) [inline firmware/config/boards/48way/board_configuration.cpp:56]: please remove dead code

File: firmware/config/boards/48way/connectors/main.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
+pins:
+    - id: E2
+      class: outputs
+      ts_name: Fan Relay (E2)
+    - id: E3
+      class: outputs
+      ts_name: Fuel Relay (E3)    
+    - id: E4
+      class: analog_inputs
+      ts_name: Launch Control (E4)
```
Comment (rusefillc) [inline firmware/config/boards/48way/connectors/main.yaml:10]: launch control is not analog_inputs

File: firmware/config/boards/48way/connectors/main.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      class: analog_inputs
+      ts_name: Battery Reference Input (A0)
+    - id: A1
+      class: analog_inputs
+      ts_name: TPS Reference Input (A1)
+    - id: A2
+      class: analog_inputs
+      ts_name: CLT Reference Input (A2)
+    - id: A3
+      class: analog_inputs
```
Comment (rusefillc) [inline firmware/config/boards/48way/connectors/main.yaml:27]: analog inputs should be using EFI_ADC_* notation not pin ID notation - please see how IAT and TPS is handled in other boards

**Review by rusefillc — CHANGES_REQUESTED** [review]
please fix analog inputs in yaml file

File: firmware/config/boards/48way/connectors/main.yaml (hunk 2 above)
Comment (sbtoonz) [reply firmware/config/boards/48way/connectors/main.yaml:10]: Having hard time referencing another board that is using LC

File: firmware/config/boards/48way/connectors/main.yaml (hunk 3 above)
Comment (sbtoonz) [reply firmware/config/boards/48way/connectors/main.yaml:27]:
```
    - id: [A2, EFI_ADC_2]
      class: [switch_inputs, analog_inputs]
      ts_name: CLT Reference Input (A2)
```

such as this format @rusefillc  ?

File: firmware/config/boards/48way/connectors/main.yaml (hunk 2 above)
Comment (rusefillc) [reply firmware/config/boards/48way/connectors/main.yaml:10]:
```
  - pin: C4
    # H144_IN_VSS
    id: F11
    class: switch_inputs
    function: 2Step Digital Input
    type: hall
    ts_name: C4 - 2Step/Digital
    color: gray
```

File: firmware/config/boards/48way/connectors/main.yaml (hunk 3 above)
Comment (rusefillc) [reply firmware/config/boards/48way/connectors/main.yaml:27]: just use the simple form
```
  - pin: B3
    id: EFI_ADC_13
    class: analog_inputs
    function: IAT Sensor Input
    ts_name: B3 - IAT
    color: orange
```

**Comment by rusefillc** [comment]
you have connector image but you do not use "pin" keys and do not have coordinates in the yaml file

please either add pin coordinates into the yaml file or remove the image

**Comment by sbtoonz** [comment]
ok request re review please @rusefillc

**Comment by sbtoonz** [comment]
> you have connector image but you do not use "pin" keys and do not have coordinates in the yaml file
> 
> please either add pin coordinates into the yaml file or remove the image

This will take me a few hours to put together but is my goal, I also realized the YML most likely isn't helpful without these kinds of things either.

**Pull request merged**